include/pagmo/algorithms/pso_gen.hpp

```diff
@@ -96,7 +98,7 @@ const int pso_gen_statics<T>::vonNeumann_neighb_diff[4][2] = {{-1, 0}, {1, 0}, {
  *
  * \endverbatim
  */
-class pso_gen : public pso_gen_statics<>
```

Comment:
preferisci non derivare ed accedere tramite namespace detail? Era cosi, poi lo avevo cambiato, mi sembrava piu corto :)

Reply:
There was a public inheritance from a class which is not even showing up in the sphinx docs, and which really looks like an internal implementation detail. Perhaps it could also be shared between the two pso implementations, Don't Repeat Yourself and all that jazz :)